var/spack/repos/builtin/packages/apple-gl/package.py

```diff
@@ -50,3 +30,16 @@ def headers(self):
     @property
     def libs(self):
         return LibraryList(join_path(self.prefix, "System/Library/Frameworks/OpenGL.framework"))
+
+
+class AppleGl(AppleGlBase):
+    """Shim package for the core OpenGL library from Apple"""
+
+    version("4.1.0")
+
+    provides("gl@4.1")
+
+    requires(
```

Comment:
Curious why this can't be in the base pacakge

Reply:
The message is slightly different